src/closure_externs.js

```diff
@@ -29,21 +29,35 @@ var HTMLTableHeaderCellElement;
 
 /**
  * Closure's NodeList is parameterized itself, there is no NodeListOf.
- * @typedef {!NodeList}
+ * @constructor
+ * @template T
+ * @extends {NodeList<T>}
  */
 var NodeListOf;
 
 /**
  * Closure models this as a plain Array.
- * @typedef {Array<string>|null}
+ * @typedef {!IArrayLike<string>|null}
```

Comment:
rather than copying parts of this file, is it possible we could use the real thing by copying from somewhere during the build?

Reply:
Not sure I follow, this file is the real thing, isn't? It's the bridge aliasing TypeScript types to Closure types, there's no other artefact that has the same functionality.

Reply:
Ah, I understand now, I thought these were duplicated from the built-in externs (eg. https://github.com/google/closure-compiler/tree/master/externs/browser)